Cargo.toml

```diff
@@ -19,13 +19,13 @@ debug = 1
 async-std = "1.10"
 async-trait = "0.1"
 bytes = "1"
+cc = "=1.0.105" # pinning version supporting rustc 1.65
```

Comment:
Why is this needed?

I think it's acceptable that msrv does not work with latest dependencies' versions if that's the reason. But preventing higher versions here could lead to resolution problems if any other library in someone's dependency tree uses newer `cc`.

Reply:
Fair but the CI won't finish otherwise. We either need to adjust the CI or keep this

Reply:
Well, good enough. I also felt free to remove the `url` pinning (for the same reason)